### PR TITLE
DR troubleshoot - sharing project with mergin maps support

### PR DESCRIPTION
**Pull request by alex-cit**
related to  #436 
Dashboard Redesign - screenshots update -  https://merginmaps.com/docs/misc/troubleshoot/

### CODE REVIEW

**Review by MarcelGeo — APPROVED** [review]

File: src/misc/troubleshoot.md [hunk 1]
```diff
@@ -40,24 +40,18 @@ Sometimes it is useful to share your project with our support team so that they
 
 To share your <MainPlatformName /> project with the support:
```
Comment (raherin) [inline src/misc/troubleshoot.md:41]: Edit text:
To share your <MainPlatformName /> project with the support team:

File: src/misc/troubleshoot.md [hunk 2]
```diff
@@ -40,24 +40,18 @@ Sometimes it is useful to share your project with our support team so that they
...
 1. Log in to <AppDomainNameLink />
-2. In the **Project** tab in the left panel, find the project that needs to be shared with the support
-   ![List of projects in Mergin Maps](./mergin-find-project.jpg "List of projects in Mergin Maps")
+2. In the **Project** tab in the left panel, find the project that needs to be shared with the support and click on it.
```
Comment (raherin) [inline src/misc/troubleshoot.md:43]: Edit text:
In the **Project** tab on the left panel, find the project that needs to be shared with the support team and click on it.

File: src/misc/troubleshoot.md [hunk 3]
```diff
@@ -40,24 +40,18 @@ Sometimes it is useful to share your project with our support team so that they
...
 
-4. Enter the email <MerginMapsEmail id="support" /> and click **Next**
-   ![Sharing Mergin Maps project with the support](./mergin-share-project-with-support.jpg "Sharing Mergin Maps project with the support")
-   
-5. Check the details in the invite form. If you want to share multiple project with the support, you may add them to the invitation. Click **Next**.
-   ![Sharing Mergin Maps project with the support](./mergin-share-project-with-support-2.jpg "Sharing Mergin Maps project with the support")
-
-6. Click **Share** to send the invitation to the support.
-   ![Sharing Mergin Maps project with the support](./mergin-share-project-with-support-3.jpg "Sharing Mergin Maps project with the support")
+4. Use the email <MerginMapsEmail id="support" /> in the sharing form and click **Share** to send the invitation to the support.
```
Comment (raherin) [inline src/misc/troubleshoot.md:53]: Edit text:
Use the email `<MerginMapsEmail id="support" />` in the sharing form and click **Share** to send the invitation to the support team.

File: src/misc/troubleshoot.md [hunk 4]
```diff
@@ -40,24 +40,18 @@ Sometimes it is useful to share your project with our support team so that they
...
+   ![Sharing Mergin Maps project with the support](./mergin-maps-share-project-with-support.jpg "Sharing Mergin Maps project with the support")
```
Comment (raherin) [inline src/misc/troubleshoot.md:54]: Edit image text:
`Sharing Mergin Maps project with the support team`

**Review by raherin — CHANGES_REQUESTED** [review]

**Pull request merged**